.github/workflows/e2e-compatibility-workflow-call.yaml

```diff
@@ -43,6 +43,7 @@ jobs:
           repository: cosmos/ibc-go
       - uses: actions/setup-go@v4
         with:
+          cache-dependency-path: "**/*.sum"
```

Comment:
this doesn't seem to be enough on its own, we can probably look into [this](https://github.com/actions/cache)

Reply:
I've had bad experiences with that but it was more than a year ago.